Sources_v3/APIClient/Endpoints/MessageEndpoints.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension Endpoint {
+    static func deleteMessage(messageId: MessageId) -> Endpoint<EmptyResponse> {
+        .init(
+            path: "messages/\(messageId)",
+            method: .delete,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: nil
+        )
+    }
+    
+    static func editMessage(userId: UserId, messageId: MessageId, text: String) -> Endpoint<EmptyResponse> {
+        .init(
+            path: "messages/\(messageId)",
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: true,
+            body: EditMessageRequestBody(clientId: userId, text: text)
+        )
+    }
```

Comment:
Do we have to use this endpoint then? Did you try it with the message-send endpoint?

Reply:
Yes, that was the very first thing I tried. When we use the send-message endpoint for editing, we get the error saying that the message with the provided `id` already exist